source/manuals/programming-languages/python/python.html.md.erb

```diff
@@ -35,119 +37,81 @@ Couple this with the fact that much of the time GDS developers are coding web ap
 have to deal with nested `JSON` objects, ORM model definitions/ queries, and error/ url
 strings and this convention begins to show its age.
 
-```
-Do not do
-
-if not models.Address.query(
-    models.Address.street_address_line_1
-    == user['address']['street_address_line_1']
-):
-    pass
-
-
-Do
-
-if not models.Address.query(models.Address.street_address_line_1 == user['address']['street_address_line_1']):
-    pass
```

Comment:
i removed this block because if you have `black` enabled, then that takes all the opinion out of which of these is better (in this case it uses the second one). i think it's still nice to include the paragraphs above as justification for _why_ we set line length to 120